spec/requests/casa_admins_spec.rb

```diff
@@ -27,7 +27,7 @@
       it "cannot access a casa admin edit page" do
         get edit_casa_admin_path(create(:casa_admin))
 
-        expect(response).to redirect_to new_user_session_path
+        expect(response).to redirect_to root_path
```

Comment:
This seems like a change in behavior. Does the `new_user_session_path` redirect to the `root_path` and if so if the root path changes is that what we want to always redirect to if you aren't authorized?

Reply:
@noahc This was reverted back by the final commit.